app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -18,12 +18,18 @@
 
     <!--[if lt IE 9]><%= javascript_include_tag "ie.js", integrity: true, crossorigin: "anonymous" %><![endif]-->
 
-    <!-- the colour used for theme-color is the standard palette $black from GOVUK Frontend -->
+    <link rel="shortcut icon" href="<%= asset_path 'favicon.ico' %>" type="image/x-icon" />
```

Comment:
Have all of these assets been moved into the gem?

Reply:
As far as I can tell the assets are now being imported from GOVUK Frontend